### DISCUSSION TITLE
What is the proper way to add Rate Limiting?

### DISCUSSION THREAD

**Discussion by lahin31** [nextauthjs/next-auth#7072] [Help]
I want to know what is the proper way to add Rate Limiting for example Login and Registration.

**Comment by issam-seghir** [comment]
did you find a solution for that ?

**Reply by lahin31** [reply]
Yes, using lru-cache.

**Reply by issam-seghir** [reply]
this is a caching library, how do you use it as rate limiter ? , can you show me an example ?
